helix-term/src/commands/lsp.rs

```diff
@@ -116,7 +116,7 @@ impl ui::menu::Item for SymbolInformationItem {
         } else {
             match self.symbol.location.uri.to_file_path() {
                 Ok(path) => {
-                    let get_relative_path = path::get_relative_path(path.as_path());
+                    let get_relative_path = path::get_relative_path(path.as_path()).unwrap();
```

Comment:
This can use the absolute path rather than panicking

Reply:
Thanks, corrected !